src/main/java/net/aufdemrand/denizen/objects/dEntity.java

```diff
@@ -2135,6 +2135,24 @@ else if (!getLivingEntity().getActivePotionEffects().isEmpty()) {
         }
 
         // <--[tag]
+        // @attribute <e@entity.target>
+        // @returns dEntity
+        // @group attributes
+        // @description
+        // Returns the target entity of the creature, if any.
+        // Note: use <n@npc.navigator.target_entity> for NPC's.
+        // -->
+        if (attribute.startsWith("target")) {
+            if (getBukkitEntity() instanceof Creature) {
+                dEntity target = new dEntity(((Creature) getLivingEntity()).getTarget());
+                if (target != null) {
+                    return target.getAttribute(attribute.fulfill(1));
+                }
+            }
+            return null;
+        }

```

Comment:
Instead of being a tag in dEntity, could you make this a property? With a mech for it too. We need to not add more dEntity stuff that we'll have to convert eventually anyway, we already have so much of it